sendrecv/gst/webrtc-sendrecv.c

```diff
@@ -114,11 +114,23 @@ handle_media_stream (GstPad * pad, GstElement * pipe, const char * convert_name,
   g_assert_nonnull (conv);
   sink = gst_element_factory_make (sink_name, NULL);
   g_assert_nonnull (sink);
-  gst_bin_add_many (GST_BIN (pipe), q, conv, sink, NULL);
-  gst_element_sync_state_with_parent (q);
-  gst_element_sync_state_with_parent (conv);
-  gst_element_sync_state_with_parent (sink);
-  gst_element_link_many (q, conv, sink, NULL);
+
+  if (convert_name == "audioconvert") {
```

Comment:
I just noticed this. String literal comparison with `==` is undefined, because it compares pointer addresses, not contents. This might work with some compilers and fail with others. You should use:

 ```c
if (g_strcmp0 (convert_name, "audioconvert") == 0) {
```

Sorry for missing this earlier!